aten/src/ATen/native/UnaryOps.cpp

```diff
@@ -181,7 +181,7 @@ IMPLEMENT_UNARY_OP_VEC(asin)
 IMPLEMENT_UNARY_OP_VEC(atan)
 IMPLEMENT_UNARY_OP_VEC(ceil)
 IMPLEMENT_UNARY_OP_VEC(cos)
-IMPLEMENT_UNARY_OP_TH(cosh)
+IMPLEMENT_UNARY_OP_VEC(cosh)
```

Comment:
Can we remove the IMPLEMENT_UNARY_OP_TH macro defined above?

Reply:
Sounds good.